src/shared-components/Body1/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { Typography } from '@mui/material';
+
+interface Body1Props {
+  children: string | JSX.Element;
+}
+
+export const Body1 = ({ children }: Body1Props) => {
```

Comment:
I think this shold be Regular1 according to the design https://www.figma.com/file/6KSlLFezGkD8uZXhuTsx3j/Montra---Expense-Tracker-UI-Kit-(Community)?type=design&node-id=216-2&mode=design&t=UkLZq6wPCcYQEqoC-0